### PR TITLE
Change product type

### PR DESCRIPTION
**Pull request by nim-odoo**

### CODE REVIEW

**Comment by nim-odoo** [comment]
robodoo rebase-ff

**Review by amoyaux — APPROVED** [review]
Shouldn't we add. "It's recommended to archive this product and create a new one". At the end of warning? Otherwise everything is good to me.

File: addons/stock_account/models/stock_move.py [hunk 1]
```diff
@@ -237,7 +237,7 @@ def _action_done(self, cancel_backorder=False):
         # Init a dict that will group the moves by valuation type, according to `move._is_valued_type`.
         valued_moves = {valued_type: self.env['stock.move'] for valued_type in self._get_valued_types()}
         for move in self:
-            if float_is_zero(move.quantity_done, precision_rounding=move.product_uom.rounding):
+            if float_is_zero(move.quantity_done, precision_rounding=move.product_uom.rounding) or move.product_id.type == 'consu':
```
Comment (amoyaux) [inline addons/stock_account/models/stock_move.py:240]: landed_cost_ok to fix the test

**Comment by nim-odoo** [comment]
robodoo r+

**Comment by d-fence** [comment]
robodoo retry

**Pull request closed without merging**